loader.py

```diff
@@ -38,6 +49,11 @@ def organize(self, notelist):
                     n["release"]["system_requirements"] = ""
             # Push unresolved tags to separate "Known Issues" list.
             for note in n["notes"]:
+                bugs_list = []
+                for b in ('bug', 'bug2', 'bug3', 'bug4', 'bug5', 'bug6', 'bug7', 'bug8', 'bug9'):
```

Comment:
Big time yuck, can we write a script to reformat this garbage as an inline list`[bug1, bug2, etc]` in the underlying data and then get rid of this?

Reply:
Yes, I have that mostly ready. the fun part is keeping the yaml ordering and comments in tact. I wanted to get this much in the repo for when Dave does releases.

Reply:
OK. And yeah, I knew that part would be a pain. :)